packages/compiler-cli/src/ngtsc/typecheck/extended/checks/uninvoked_function_in_event_binding/index.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  AST,
+  ASTWithSource,
+  Call,
+  Chain,
+  Conditional,
+  ParsedEventType,
+  ParseSourceSpan,
+  PropertyRead,
+  SafeCall,
+  SafePropertyRead,
+  TmplAstBoundEvent,
+  TmplAstNode,
+} from '@angular/compiler';
+import ts from 'typescript';
+
+import {ErrorCode, ExtendedTemplateDiagnosticName} from '../../../../diagnostics';
+import {NgTemplateDiagnostic, Symbol, SymbolKind} from '../../../api';
+import {TemplateCheckFactory, TemplateCheckWithVisitor, TemplateContext} from '../../api';
+
+/**
+ * Ensures that function in event bindings are called. For example, `<button (click)="myFunc"></button>`
+ * will not call `myFunc` when the button is clicked. Instead, it should be `<button (click)="myFunc()"></button>`.
+ * This is likely not the intent of the developer. Instead, the intent is likely to call `myFunc`.
+ */
+class UncalledFunctionInEventBindingSpec extends TemplateCheckWithVisitor<ErrorCode.UNINVOKED_FUNCTION_IN_EVENT_BINDING> {
```

Comment:
For consistency:

```suggestion
class UninvokedFunctionInEventBindingSpec extends TemplateCheckWithVisitor<ErrorCode.UNINVOKED_FUNCTION_IN_EVENT_BINDING> {
```


(although my preferred wording would have been `Uncalled`, I think)

Reply:
I like **uncalled** too, but we have the signal not invoked diagnostic too, which uses the same terminology. So keeping it as is for now.